docs/usage/verticles/EntityVerticle.md

```diff
@@ -0,0 +1,35 @@
+# EntityVerticle
+
+# Create
+
+A `POST` request will be used to create a new entity. In this case, the `Future<EntityWrapper> createData(DataQuery query, DataContext context)` method of the responsible `EntityVerticle` will be invoked.
+According to the best-practices of REST-API, the response of such a POST-request should
+* contain a `Location`-header with the location of the newly created entity
+* return the entity representation in the response body
+
+To achieve this, a `EntityVertile` should in the `createEntity`-method
+* set the `Location` in the response context data
+```
+context.responseData().put("Location", /io.neonbee.test3.TestService3/TestCars(/unique-id));
+```
+* return a filled `EntityWrapper`
+```
+return Future.succeededFuture(new EntityWrapper(TEST_ENTITY_SET_FQN, createEntity(ENTITY_DATA_1));
+```
+In this case, a 201 status code with the entity JSON representation will be returned in the HTTP-response.
```

Comment:
the representation can be JSON or XML depending on the request params.

Reply:
Removed JSON to make it format neutral